bin/start-pgbouncer-as-service

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+
+set +e
+
+export PGBOUNCER_PROCESS_REGEX="[v]endor/pgbouncer/bin/pgbouncer"
+export PGBOUNCER_BUILD_PACK_PID
+
+function inf() {
+  echo -e "INFO:  $* "
+}
+
+function err() {
+   echo -e "ERROR: $* "
+}
+
+function is-enabled() {
+  ( shopt -s extglob nocasematch
+    [[ $1 == @(1|true|yes|on) ]]
+  )
+}
+
+function pgbouncer-current-pid() {
+  /bin/ps -eopid,args | grep -E "${PGBOUNCER_PROCESS_REGEX}" | awk '{print $1}'
+}
+
+function is-pgbouncer-running() {
+  local current_pid
+  current_pid=$(pgbouncer-current-pid)
+
+  [[ -n ${current_pid} && ${current_pid} -gt 0 ]] && {
+    inf "pgbouncer-is-running: pid=${current_pid}"
+    return 0
+  }
+
+  inf "pgbouncer-is-not-running"
+  return 1
+}
+
+function is-pgbouncer-service-enabled() {
+  if ! is-enabled "${PGBOUNCER_ENABLED}"; then
+    inf pgbouncer-is-disabled
+    return 1
+  fi
+  return 0
+}
+
+function main() {
+  is-pgbouncer-service-enabled || return 1
+
+  local stderr
+  stderr="$(mktemp)"
+
+  [[ -x bin/start-pgbouncer ]] && bin/start-pgbouncer /bin/bash -c "while true; do sleep 10000; done" 2>"${stderr}" &
+
+  sleep 1
+
+  is-pgbouncer-running || return 1
+
+  PGBOUNCER_BUILD_PACK_PID=$(pgbouncer-current-pid)
+  export PGBOUNCER_BUILD_PACK_PID
+
+  if [[ -n ${PGBOUNCER_BUILD_PACK_PID} && ${PGBOUNCER_BUILD_PACK_PID} -gt 0 ]] ; then
+    set +e
+    mkdir -f tmp/pids || true
```

Comment:
`-f` doesn't seem to be a valid flag for `mkdir`. Both the logs on Heroku and my own local machine are complaining about it. Did you want `-p` instead? 

<img width="389" alt="Screenshot 2023-11-28 at 10 21 14 AM" src="https://github.com/healthsherpa/heroku-buildpack-pgbouncer/assets/362585/b3cd7690-d0f1-414b-8085-d7eda73595d5">